### PR TITLE
Update the docs to hint database timeouts

### PR DESCRIPTION
**Pull request by cleptric**
This came up in #13

### CODE REVIEW

File: docs/configuration.rst [hunk 1]
```diff
@@ -20,3 +20,8 @@ config stanza:
 Note that the config array is passed as settings to the queueing engine. Please
 refer to the Queuesadilla `docs <http://josegonzalez.viewdocs.io/php-queuesadilla/>`_
 for more information on how each engine can be configured.
+
+**Please Note:**
+
+If your job callback is performing a database involved operation, you should check if the CakePHP database connection is still alive.
+It can happen that your database connection timed out, when no jobs were acknowledged for a long time.
```
Comment (josegonzalez) [inline docs/configuration.rst]: Can you show code to create to reset the connection?

File: docs/configuration.rst [hunk 2]
```diff
@@ -25,3 +25,11 @@ for more information on how each engine can be configured.
...
 If your job callback is performing a database involved operation, you should check if the CakePHP database connection is still alive.
 It can happen that your database connection timed out, when no jobs were acknowledged for a long time.
+
+.. code:: php
+
+    // Disconnect the connection fist...
+    ConnectionManager::get('default')->disconnect();
+
+    // ...and renew it inside a Table class
+    $this->connection(ConnectionManager::get('default'));
```
Comment (josegonzalez) [inline docs/configuration.rst]: Any reason why you didn't just show an example using the event system like you did in the comment on the other issue?

Comment (cleptric) [inline docs/configuration.rst]: Oh, I just wanted to show a more "generic" way. But I can change to my initial comment

File: docs/configuration.rst [hunk 3]
```diff
@@ -20,3 +20,26 @@ config stanza:
...
+    public function getWorker($engine, $logger)
+    {
+        $worker = parent::getWorker($engine, $logger);
+
+        $worker->attachListener('Worker.job.success', function ($event) {
+            ConnectionManager::get('default')->disconnect();
+        });
+        $worker->attachListener('Worker.job.failure', function ($event) {
+            ConnectionManager::get('default')->disconnect();
+        });
```
Comment (josegonzalez) [inline docs/configuration.rst]: Is there a way to hook up these events to the global event manager? If so, that would be better, as you can place that in your app's `config/bootstrap.php`. If not, worth implementing that...

Comment (cleptric) [inline docs/configuration.rst]: We discussed that in #7
I'll try to implement this, this or next week

**Pull request merged**